recipes/openmesh/build.sh

```
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+else
+  SO_EXT='.so'
+fi
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_APPS=OFF \
+  -DOPENMESH_PYTHON_VERSION=${PY_VER} \
+  -DOPENMESH_BUILD_PYTHON_UNIT_TESTS=ON \
+  ..
+
+make install -j${CPU_COUNT} VERBOSE=1
+
+# osx, py34: ***Exception: SegFault in test_load_obj_with_material (test_read_write_obj.ReadWriteOBJ)
```

there not much output except from this, nor traceback, and I dont have access to a mac, i'll call ctest and let it fail for you to see.